components/pluralrules/src/bin/generate_res.rs

```diff
@@ -0,0 +1,19 @@
+use icu_pluralrules::data::io::json::get_resource;
```

Comment:
Note, I skipped the test data files in review. I suppose those are autogenerated.